src/os/bluestore/bluestore_types.cc

```diff
@@ -752,12 +752,14 @@ void bluestore_lextent_t::encode(bufferlist& bl) const
   small_encode_signed_varint(blob, bl);
   small_encode_varint_lowz(offset, bl);
   small_encode_varint_lowz(length, bl);
+  small_encode_varint_lowz(blob_depth, bl);
```

Comment:
no need to use _lowz encoding here as it's intended for values that tend to have low bits set to zero. Suggest to use small_encode_varint or even replace blob_depth type with uint8_t (if level 256 is enough in any case) that requires no small encoding at all.